portfolio/src/main/java/com/google/sps/data/CommentDatabase.java

```diff
@@ -29,11 +31,10 @@ public CommentDatabase(DatastoreService datastore) {
 
   private void getCommentsFromQuery() {
     PreparedQuery storedCommentsQuery = datastore.prepare(COMMENT_QUERY);
-    this.comments = new ArrayList<Comment>();
-    for (Entity entity : storedCommentsQuery.asIterable()) {
-      Comment comment = new Comment((String) entity.getProperty(AUTHOR_QUERY_STRING), (String) entity.getProperty(VALUE_QUERY_STRING));
-      this.comments.add(comment);
-    }  
+    this.comments = StreamSupport.stream(storedCommentsQuery.asIterable().spliterator(), false)
```

Comment:
include comment when it's unclear what a boolean does. i.e " ...spliterator, /* enableCoolFeature */ false)